README.md

```diff
@@ -385,6 +385,7 @@ Services to securely store your Docker images.
 * [FaaS](https://github.com/alexellis/faas) - Docker Serverless/Functions as a Service (on Docker Swarm). By [@alexellis](https://github.com/alexellis/)
 * [Funker](https://github.com/bfirsh/funker-example-voting-app) - Functions as Docker containers example voting app. By [@bfirsh](https://github.com/bfirsh)
 * [IronFunctions](https://github.com/iron-io/functions) - The serverless microservices platform FaaS (Funcitons as a Service) which uses Docker containers to run Any language or AWS Lambda functions
+* [SCAR](https://github.com/grycap/scar) - Serverless & Container Aware Architectures (SCAR) by [Grycap](https://github.com/grycap)
```

Comment:
A bit more description would be nice, like as you did on the PR comment.

`Serverless & Container Aware Architecture (SCAR) is a serverless framework that allows easy deployment and execution of containers in Serverless environments i.e. Lambda`

And `@` symbol is missing on the author's GitHub link.